src/qtism/data/storage/xml/marshalling/MarshallerFactory.php

```diff
@@ -399,34 +405,41 @@ public function createMarshaller($object, array $args = [])
                 // Look for a mapping entry.
                 if ($object instanceof DOMElement && $this->hasMappingEntry($qtiClassName, $object->namespaceURI)) {
                     $class = new ReflectionClass($this->getMappingEntry($qtiClassName, $object->namespaceURI));
-                } elseif ($object instanceof QtiNamespaced && $this->hasMappingEntry($qtiClassName, $object->getTargetNamespace())) {
+                } elseif ($object instanceof QtiNamespaced && $this->hasMappingEntry(
+                        $qtiClassName,
+                        $object->getTargetNamespace()
+                    )) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place